reader.go

```diff
@@ -138,11 +138,14 @@ func (r *Reader) Read() (File, error) {
 		lineLength := len(line)
 
 		switch {
-		case r.lineNum == 1 && lineLength > RecordLength && lineLength%RecordLength == 0:
-			if err := r.processFixedWidthFile(&line); err != nil {
+		case r.lineNum == 1 && lineLength > RecordLength:
+			extraChars := lineLength % RecordLength
+			if extraChars != 0 {
+				r.errors.Add(r.parseError(NewRecordWrongLengthErr(lineLength)))
+				r.errors.Add(fmt.Errorf("%d extra characters in ACH file", extraChars))
```

Comment:
Can we add a test verifying this error message?

Reply:
sure thing